mod-source-record-storage-server/src/main/java/org/folio/services/util/EventHandlingUtil.java

```diff
@@ -100,8 +105,7 @@ public static String createSubscriptionPattern(String env, String eventType) {
   }
 
   public static KafkaProducer<String, String> createProducer(String eventType, KafkaConfig kafkaConfig) {
-    String producerName = eventType + "_Producer";
-    return KafkaProducer.createShared(Vertx.currentContext().owner(), producerName, kafkaConfig.getProducerProps());
+    return new SimpleKafkaProducerManager(Vertx.currentContext().owner(), kafkaConfig).createShared(eventType);
```

Comment:
should the manager here be a singleton as well?

Reply:
I do not think so. Because since we declaring it with private final and initializing it within constructor, producerManager is used in util method, which is util method. Also, we have got to pass it kafkaConfig property to producerManager during creation. So I leave it as it is)